driver/src/util.rs

```diff
@@ -38,18 +38,31 @@ pub fn hash_consistency_check(
     Ok(())
 }
 
+pub enum ProcessingState {
```

Comment:
Yes, good point. We should add it when it becomes a possibility in the code.

Reply:
One note about this, if we add the state Ben is suggesting in the future, in the matching of `withdraw_driver.rs:26`, you won't get a compiler error, cause it was handled the default to join the `AcceptsBids` and `AcceptsSolution`. 

It might be a good idea for this cases, match the specific cases, so if you add items to the enum, you are forced to handle the case. 

I think you can target multiple cases by using `AcceptsBids | AcceptsSolution => ...`